doesnt build

2025-02-15T00:02:52.851Z	Cloning repository...
2025-02-15T00:02:53.729374Z	From https://github.com/KdogDevs/shift2stream-website
2025-02-15T00:02:53.729867Z	 * branch            ccf460933e19f3c4b953c19506505d7d57efb605 -> FETCH_HEAD
2025-02-15T00:02:53.729981Z	
2025-02-15T00:02:53.811969Z	HEAD is now at ccf4609 npm(deps): bump the npm_and_yarn group with 2 updates
2025-02-15T00:02:53.812352Z	
2025-02-15T00:02:53.894894Z	
2025-02-15T00:02:53.895329Z	Using v2 root directory strategy
2025-02-15T00:02:53.915812Z	Success: Finished cloning repository files
2025-02-15T00:02:55.471971Z	Checking for configuration in a Wrangler configuration file (BETA)
2025-02-15T00:02:55.47304Z	
2025-02-15T00:02:56.590339Z	No wrangler.toml file found. Continuing.
2025-02-15T00:02:56.657591Z	Detected the following tools from environment: npm@9.6.7, nodejs@18.17.1
2025-02-15T00:02:56.65831Z	Installing project dependencies: npm clean-install --progress=false
2025-02-15T00:02:59.394973Z	npm WARN deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
2025-02-15T00:02:59.819102Z	npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
2025-02-15T00:02:59.966932Z	npm WARN deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
2025-02-15T00:03:00.819007Z	npm WARN deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
2025-02-15T00:03:00.880562Z	npm WARN deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
2025-02-15T00:03:02.022452Z	npm WARN deprecated three-mesh-bvh@0.7.8: Deprecated due to three.js version incompatibility. Please use v0.8.0, instead.
2025-02-15T00:03:03.296389Z	npm WARN deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.
2025-02-15T00:03:09.869898Z	
2025-02-15T00:03:09.870162Z	added 457 packages, and audited 458 packages in 13s
2025-02-15T00:03:09.870309Z	
2025-02-15T00:03:09.87042Z	74 packages are looking for funding
2025-02-15T00:03:09.870532Z	  run `npm fund` for details
2025-02-15T00:03:09.87178Z	
2025-02-15T00:03:09.872054Z	found 0 vulnerabilities
2025-02-15T00:03:09.888316Z	Executing user command: npm run build
2025-02-15T00:03:10.445718Z	
2025-02-15T00:03:10.445992Z	> shift2stream-new@0.0.1 build
2025-02-15T00:03:10.446108Z	> tsc -b && vite build
2025-02-15T00:03:10.446183Z	
2025-02-15T00:03:13.233574Z	src/pages/my-velo-tv.tsx(6,10): error TS6133: 'isHovered' is declared but its value is never read.
2025-02-15T00:03:13.43065Z	Failed: Error while executing user command. Exited with error code: 2
2025-02-15T00:03:13.441684Z	Failed: build command exited with code: 1
2025-02-15T00:03:14.45279Z	Failed: error occurred while running build command

--- a/src/pages/my-velo-tv.tsx
+++ b/src/pages/my-velo-tv.tsx
@@ -1,10 +1,8 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { motion } from 'framer-motion';
 import '../styles/fonts.css';
 
 const MyVeloTV: React.FC = () => {
-  const [isHovered, setIsHovered] = useState(false);
-
   const service = {
     name: 'MyVelo TV',
     // Use the primary logo from the public folder
@@ -59,8 +57,6 @@ const MyVeloTV: React.FC = () => {
             rel="noopener noreferrer"
             className="block bg-white rounded-2xl shadow-2xl overflow-hidden transform transition-all duration-300 ease-in-out"
             whileHover={{ scale: 1.05 }}
-            onHoverStart={() => setIsHovered(true)}
-            onHoverEnd={() => setIsHovered(false)}
           >
             <div className="p-8">
               {/* Replace in-card header with product image */}

